remotespark/utils/configuration.py

```diff
@@ -89,14 +91,38 @@ def session_configs():
     return {}
 
 
-@_override
+def _credentials_override(f):
+    """A decorator that provide special handling for credentials. It still calls _override().
+    If 'base64_password' in config is set, it will base64 decode it and returned in return value's 'password' field.
+    If 'base64_password' is not set, it will fallback to to 'password' in config.
+    """
+    def ret():
+        credentials = _override(f)()
+        base64_decoded_credentials = {k: credentials.get(k) for k in ('username', 'password', 'url')}
```

Comment:
Doesn't `get` throw if there's no key? If that's the case, then the possible exception has to be wrapped in an expected exception so that the problem is properly displayed to the user.